### DISCUSSION TITLE
Select only part of fields

### DISCUSSION THREAD

**Discussion by YoshieraHuang** [SeaQL/sea-orm#304] [Q&A] [answered]
Hi guys, can `sea-orm` select only part of  fields in `Model` rather than the whole `Model`?

**Comment by billy1624** [comment, marked as answer]
Hi @YoshieraHuang, yes you can!

```rust
// Selecting the name column only
assert_eq!(
    cake::Entity::find()
        .select_only()
        .column(cake::Column::Name)
        .build(DbBackend::Postgres)
        .to_string(),
    r#"SELECT "cake"."name" FROM "cake""#
);
```

And, you can derive `FromQueryResult` if want to select partial fields into custom struct.

https://github.com/SeaQL/sea-orm/blob/6a0db92c8b194e3bccaaa2f7a8fe136b96b0e088/tests/relational_tests.rs#L663-L689

**Comment by YoshieraHuang** [comment]
Uhh @billy1624 , I find this at `Advanced Queries/Custom Select` in docs later. Thank you for your kind reply.
